docs/sdk/src/reference_docs/frame_currency.rs

```diff
@@ -1,8 +1,55 @@
-//! FRAME Currency Abstractions and Traits
+//! # FRAME Currency Abstractions and Traits
 //!
-//! Notes:
+//! Currency related traits in FRAME provide standardized interfaces for implementing various
+//! currency functionalities. The fundamental reason to have these traits is to allow pallets to
+//! utilize a token without relying on the implementation detail. Or else we could have a perfectly
+//! fine runtime without these traits.
+//! These traits enable developers to create, transfer, and manage different forms of digital assets
```

Comment:
> different forms of digital assets

Nit but the first thing I thought when I read this was NFTs and I believe the `Currency` traits are not the best abstractions.